commit 69d2920745d3ac11a44b5d9ce3b07e8eb1d07b25
Author: Fabio M. Di Nitto <fdinitto@redhat.com>
Date:   Tue May 15 13:53:28 2012 +0200

    cman init: allow sysconfig/cman to pass options to dlm_controld
    
    DLM_CONTROLD_OPTS="" can now be used to pass startup options to the
    daemon.
    
    Resolves: rhbz#821016
    
    Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>
    Reviewed-by: Christine Caulfield <ccaulfie@redhat.com>

diff --git a/cman/init.d/cman.in b/cman/init.d/cman.in
index 1268b59..f345bcc 100644
--- a/cman/init.d/cman.in
+++ b/cman/init.d/cman.in
@@ -107,6 +107,9 @@ fi
 # no (default)          | cman will not start sshd
 [ -z "$CMAN_SSHD_START" ] && CMAN_SSHD_START=no
 
+# DLM_CONTROLD_OPTS -- allow extra options to be passed to dlm_controld daemon.
+[ -z "$DLM_CONTROLD_OPTS" ] && DLM_CONTROLD_OPTS=""
+
 # FENCE_JOIN_TIMEOUT -- seconds to wait for fence domain join to
 #     complete.  If the join hasn't completed in this time, fence_tool join
 #     exits with an error, and this script exits with an error.  To wait
@@ -615,7 +618,7 @@ stop_fenced()
 
 start_dlm_controld()
 {
-	start_daemon dlm_controld || return 1
+	start_daemon dlm_controld "$DLM_CONTROLD_OPTS" || return 1
 
 	if [ "$INITLOGLEVEL" = "full" ]; then
 		ok
diff --git a/cman/init.d/cman.init.defaults.in b/cman/init.d/cman.init.defaults.in
index 05c3bc7..d6d010c 100644
--- a/cman/init.d/cman.init.defaults.in
+++ b/cman/init.d/cman.init.defaults.in
@@ -31,6 +31,9 @@
 # no (default)          | cman will not start sshd
 #CMAN_SSHD_START=no
 
+# DLM_CONTROLD_OPTS -- allow extra options to be passed to dlm_controld daemon.
+#DLM_CONTROLD_OPTS=""
+
 # FENCE_JOIN_TIMEOUT -- seconds to wait for fence domain join to
 #     complete.  If the join hasn't completed in this time, fence_tool join
 #     exits with an error, and this script exits with an error.  To wait
